arch/arm/src/samv7/Kconfig

```diff
@@ -1445,6 +1449,11 @@ config SAMV7_USART0_1WIREDRIVER
 	bool "1-Wire driver"
 	select SAMV7_1WIREDRIVER
 
+config SAMV7_USART0_LIN_SOCKET
+	bool "SocketLIN driver"
+	select SAMV7_LIN_SOCKET
+	depends on NET_CAN
```

Comment:
Should it be better if we had a NET_LIN ? How does Linux implement SocketLin?

Reply:
Linux use sLIN and it is described as
```
slLIN is TTY line discipline implemented for GNU/Linux operating system that handles the interchange of LIN messages between the underlying UART driver and the CAN bus subsystem.

This way, the same API as the one used for CAN networking can be used for communication with LIN devices.
```
So also sharing a CAN layer. I do not see any reason for introducing new name that duplicates existing unless there is some strong opinion about that from the community.

Reply:
Actually "SocketLIN" is something that I came up with, but maybe it is better to name it "LIN over SocketCAN".

Reply:
Yes, I think is makes more sense, what do you think?

Reply:
I like SocketLIN, it's a nice abbreviation, but I agree "LIN over SocketCAN" or something similar provides more clarity. Also please consider adding `---help---` part to `SAMV7_USARTx_LIN_SOCKET` options with some description.

Reply:
Please check if new variant is more clear